spec/models/task_spec.rb

```diff
@@ -37,6 +37,23 @@
         expect(task.active_submission).to eq(completed_submission)
       end
     end
+
+    context 'there are multiple completed submissions' do
+      let!(:previous_completed_submission) do
```

Comment:
Not a blocker, but we didn't include this check as there should never be more than one completed submission against a task. We will mark old completed submissions as 'replaced' in a transaction when making a new completed submission.

Reply:
Ah, ok, I'll remove this spec then.